test/filter_test.cc

```diff
@@ -75,7 +76,7 @@ int main(int argc, char* argv[]) {
 
   // Make sure we ran all of the tests
   const size_t count = test_reporter.GetCount();
-  const size_t expected = (argc == 2) ? std::stoul(argv[1]) : count;
+  const size_t expected = (argc == 2) ? std::atol(argv[1]) : count;
```

Comment:
Mixing signed and unsigned is going to cause a mess on Windows.  I don't think we should do this at a time when other PRs are trying to fix signed/unsigned confusion.